jaco_moveit_config/config/joint_limits.yaml

```diff
@@ -24,25 +22,19 @@ joint_limits:
     has_acceleration_limits: false
     max_acceleration: 0.6
   jaco_joint_4:
```

Comment:
Joint 4,5,6 are continuous, right? Did you find a reason to make them have limits?